src/main/webapp/app/exam/exam-scores/exam-scores.component.html

```diff
@@ -304,10 +304,28 @@ <h4>
             </tbody>
         </table>
 
-        <div style="display: block">
+        <div #containerRef style="display: block">
```

Comment:
same here (as from my experience, you can replace every single inline style with bootstrap or css (if bootstrap did not work out or should be reused multiple times/is too complex)

Reply:
Done in ddade2e as well